django_x509/utils.py

```diff
@@ -1,12 +1,2 @@
-import sys
-
-import six
-
-
 def bytes_compat(string, encoding='utf8'):
```

Comment:
let's go one step further, this utility function is not useful anymore, so just remove it and use `bytes(argument)` where it was used (looks it's only used in `models.py`)